packages/manager/modules/manager-pci-common/src/components/tiles-input-choice/TilesInputChoice.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { TileChoice } from './TileChoice.component';
+
+interface Item {
+  id: string;
+  isDisabled?: boolean;
+}
+
+export interface TileInputChoiceProps<T extends Item> {
+  items: T[];
+  selectedItem?: T;
+  onSelectItem?: (T: Item) => void;
+  columnsCount?: 1 | 2 | 3 | 4 | 5 | 6;
+  isSubmitted?: boolean;
+  children: (item: T, isSelected: boolean) => JSX.Element;
+}
+
+export function TileInputChoice<T extends Item>({
```

Comment:
This is not specific to pci and should be moved to MRC

Reply:
i think it's specific to pci ui, other universes do not use this type of selector